blog/2020-05-11-version-0.5.0-released.md

```diff
@@ -0,0 +1,86 @@
+---
+id: version-0.5.0-released
+title: Version 0.5.0 released
+author: Beanow
+author_url: https://github.com/Beanow
+author_image_url: https://avatars3.githubusercontent.com/u/497556?v=4&s=200
+---
+
+It's been months since the last versioned release, so there's a lot new to
+v0.5.0!
+
+You can find the release [on GitHub][GitHub release] or as a [Docker image].
+More details on how to use them can be found in the [installation guide].
+
+Since this is a large release, I'll only touch on several major additions here.
+
+### Discourse plugin
+
+Discourse forums can be analyzed with this release. And from using it so far, I
+think it's a really interesting plugin.
+
+Discourse forums by default require no authentication to be set up. Meaning you
+can load most Discourse forums out there and explore them with SourceCred, no
+approval required. The [plugin setup guide][discourse setup] walks through the
+commands to do this.
+
+Much thanks to the straightforward 💚 likes on Discourse, the out-of-the-box
+settings tend to be quite reasonable. Most of the tweaking involves balancing
+the Discourse weight relative to other platforms.
+
+### Initiatives
+
+[Initiatives] are added to SourceCred by a new plugin of the same name.
```

Comment:
Not quite sure what you mean by Initiatives are added to SC by a new plugin of the same name, but maybe that's bc I'm not familiar with the technical side of things.

Reply:
Initiatives (the concept) are added to SourceCred by the Initiatives plugin (concretely). I was wondering if I was stretching it with "plugin of the same name". Open to alternatives :]